<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sequence_build</class>
 <widget class="QWidget" name="Sequence_build">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequence - FILE</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="3,0,0,0" columnstretch="0,0,2,0,0,0,1">
   <item row="0" column="0" colspan="5">
    <widget class="QListView" name="listSequence">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushOpen">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QCommandLinkButton" name="Button_AbortSequence">
     <property name="text">
      <string>ABORT Sequence</string>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QTreeWidget" name="treeOptions">
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>Individual setpoints</string>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
      </property>
      <item>
       <property name="text">
        <string>Set Temperature</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Set Field</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wait</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Shutdown Temperature Control</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Measurements</string>
      </property>
      <property name="flags">
       <set>ItemIsUserCheckable|ItemIsEnabled|ItemIsTristate</set>
      </property>
      <item>
       <property name="text">
        <string>Resistivity vs Temperature</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsEnabled</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Resistivity vs Field</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hall-resistivity vs Temperature</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable</set>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>Miscellaneous</string>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled|ItemIsTristate</set>
      </property>
      <item>
       <property name="text">
        <string>Change Data File</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable</set>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Chain Sequence</string>
       </property>
       <property name="flags">
        <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
       </property>
      </item>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="pushSaving">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushBrowse">
     <property name="text">
      <string>Save As</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QCommandLinkButton" name="Button_RunSequence">
     <property name="text">
      <string>RUN Sequence</string>
     </property>
     <property name="description">
      <string>Save sequence to file and run it</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="lineFileLocation"/>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushClear">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineFileLocation</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
